/**:
  ros__parameters:
    roadmap:
      # === High level settings === #
      debug_output: false # Print debug information
      update_frequency: 1. # (Hz) Publish map information at this rate
      map_package_name: "roadmap" # Package where the map is
      # map_file_name: 'maps/straight.xml'  # Name of the map file
      map_file_name: "maps/test_map.xml" # Name of the map file
      # map_file_name: 'maps/test_map_rotated.xml'  # Name of the map file
      # map_file_name: "maps/test_map.yaml"
      # map_file_name: 'maps/test_map.osm'

      # === Spline settings === #
      spline:
        fit_clothoid: true # Fits a clothoid before fitting cubic splines if TRUE
        minimum_waypoint_distance: 0.25 #0.25  # Ignore waypoints closer than this distance together
        spline_sample_distance: 4.0 #6.0 # Distance between sampled waypoints on the spline
        clothoid_point_per_xm: 1.0 # Distance forced between points on the Clothoid

      velocity:
        max: 2.5 #2.5 # [m/s] Maximum velocity

      # === Visual settings === #
      scale: 1.0 # Scale of the visuals

      # === Topics === #
      # Topic to read waypoints from INSTEAD of using a predefined map
      external_waypoint_topic: "/roadmap/input"
      autoware:
        update_interval: 100000. # [s] Seconds between route refreshes
        forward_route: 2000. # [m] how far the route should be computed ahead of the vehicle
        backward_route: 10. # [m] how far the route should be computed behind the vehice
        include_other_lane: true
        remove_corners: true
